### ISSUE TITLE
Addon: Rendertime doesn't work anymore

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#6691]
### Expected behavior
Rendertime should show reasonable details at the bottom of the page.

### Actual behavior
Rendertime shows nonsense.

`Database: 0/0, Network: 0, Rendering: 0, Session: 0, I/O: 0, Other: 1550515227.71, Total: 1550515227.71`

### Steps to reproduce the problem
Enable Rendertime addon on current develop branch and look at the values e.g. in your admin panel.

Part of my config (INI style):
```
worker_debug = true
profiler = true
config_adapter = preload

```
### Friendica version you encountered the problem
2019.03-dev

### Friendica source (git, zip)
git

### PHP version
7.2.15

### SQL version
10.1.38 MariaDB

**Comment by AlfredSK** [comment]
Sorry for the noise. I'm obviously too stupid to run git pull properly.
There was an update of the rendertime addon.
Now it's working again.

Closing.

**Comment by annando** [comment]
I'm having the same issue as well. I updated both core and addons and don't have any profiler details anymore.

This should be activated with this config value in ```config/addon.config.php```:
```
        'rendertime' => [
                'callstack' => true
        ],
```

**Comment by annando** [comment]
The origin of the problem is, that the addon is still using ```$a->callstack``` which doesn't exist anymore:
https://github.com/friendica/friendica-addons/blob/develop/rendertime/rendertime.php#L52

I don't know what the preferred way now is to get this information. I only learned that with this type of class orientated coding mostly the easy way isn't the one that is chosen (although I mostly don't know why). So I would like to delegate this task to @nupplaphil or @MrPetovan.

**Comment by nupplaphil** [comment]
Caused by #6673 

The Profiler currently can only handle Logging the Profiling data. But I think it would be possible to replace `Profiler::saveLog($message)` with `Profiler::saveLog($message, &$output = null)`. If output is null, it saves to the log, otherwise to the `$output`.

Than you can replace the whole callstack-function in the addon with `$a->getProfiler()->saveLog('rendertime', $o);`

**Comment by annando** [comment]
```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Friendica\Util\Profiler::saveLog() must implement interface Psr\Log\LoggerInterface, string given, called in /path/to/friendica/addon/rendertime/rendertime.php on line 51 and defined in /path/to/friendica/src/Util/Profiler.php:128
```

**Comment by nupplaphil** [comment]
There was a change in #6641 in the method-signature, so it wasn't exactly as described at https://github.com/friendica/friendica/issues/6691#issuecomment-465575451

**Comment by annando** [comment]
It still don't work here, although I updated both core and addons.

**Comment by nupplaphil** [comment]
Do you change it in the config file or in the db? As far as I could see, it was only possible with config files. Per db it wouldn't work.. I will check it (again)

**Comment by annando** [comment]
I guess I know the reason. Lately I moved the configuration stuff for the addons to ```addon.config.php``` - but it seems as if this file isn't read at all.

**Comment by nupplaphil** [comment]
It should be loaded here:
https://github.com/friendica/friendica/blob/d2e7828bda6884e5d727fbe57c6e21ec73200775/src/App.php#L364

Thanks for the hint.

**Comment by annando** [comment]
When I see it right, then the config setting from the addon config file was known directly in the addon - but the new coding in the core don't know it, when it is only in that file.

**Comment by MrPetovan** [comment]
Closed via #6725
